### DISCUSSION TITLE
Issue with deploying blog

### DISCUSSION THREAD

**Discussion by jtgit2021** [webexpe13/blog-template-using-nextjs-typescript-tailwindcss#10] [Q&A]
Hello,

I am trying to export and deploy the blog on firebase but I am having issues.

What I did;

https://nextjs-simple-blog-template.web.app/tutorial/how-to-setup-blog/
https://nextjs-simple-blog-template.web.app/tutorial/how-to-deploy-blog/
From 1) I could do "npm start" and see that the website is running without issues on my local host.
At 2), after stopping the server and entering "npm run out" I get the error below:

info - Checking validity of types

Failed to compile.

./src/components/Navbar/Centered.tsx
120:13 Error: Missing "key" prop for element in iterator react/jsx-key

./src/components/Navbar/NavSideBar.tsx
57:33 Error: Missing "key" prop for element in iterator react/jsx-key

./src/components/Navbar/SimpleNavbar.tsx
50:17 Error: Missing "key" prop for element in iterator react/jsx-key

./src/components/Slider/index.tsx
26:25 Warning: img elements must have an alt prop, either with meaningful text, or an empty string for decorative images. jsx-a11y/alt-text

Any suggestions?

**Comment by webexpe13** [comment]
Hey,
for these three
```
./src/components/Navbar/Centered.tsx
120:13  Error: Missing "key" prop for element in iterator  react/jsx-key

./src/components/Navbar/NavSideBar.tsx
57:33  Error: Missing "key" prop for element in iterator  react/jsx-key

./src/components/Navbar/SimpleNavbar.tsx
50:17  Error: Missing "key" prop for element in iterator  react/jsx-key 
```

I guess i might have missed the key prop which we need to add while looping through an array and displaying elements.
example
```
<p props={...} key={<unique id>}>text</p>
```

You can ignore this as it's just a warning, if you  want to get rid of this, just add alt text to the image tag,
./src/components/Slider/index.tsx
26:25  Warning: img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.  jsx-a11y/alt-text

If u need any more help let me know and share the github repo with me, i will ceata branch for these bug fix

And thank you for using this template, check out https://github.com/webexpe13/nextjs-tailwind-mdx-blog-template this one too, it's made using MDX and it's more clean.

Regards,
Mayur Nalwala from Webexpe.
